reference/conanfile/running_and_output.rst

```diff
@@ -0,0 +1,55 @@
+.. _reference_conanfile_output:
+
+
+Running and output
+==================
+
+Output text from recipes
+------------------------
+
+Use the ``self.output`` attribute to output text from the recipes. Do **not** use Python's ``print()`` function.
+The ``self.output`` attribute has the following methods to express the level of the printed message:
+
+..  code-block:: python
+
+   trace(msg)
+   debug(msg)
+   verbose(msg)
+   status(msg)
+   info(msg)
+   highlight(msg)
+   success(msg)
+   warning(msg)
+   error(msg)
+
+   title(msg)
+   subtitle(msg)
+
+   flush()
```

Comment:
Probably hide the ``title, subtitle`` ones, those are reserved for Conan main flow. Probably remove ``flush()`` too